reo/scenario.py

```diff
@@ -231,16 +303,40 @@ def setup_pv(pv_dict, latitude, longitude, time_steps_per_hour):
             ModelManager.updateModel('WindModel', tmp, run_uuid)
             # TODO: remove the need for this db call by passing these values to process_results.py via reopt.jl
 
+
+        if inputs_dict["Site"]["CHP"].get("prime_mover") is not None or \
+                inputs_dict["Site"]["CHP"].get("max_kw", 0.0) > 0.0:
+            if boiler is not None:
+                steam_or_hw = boiler.existing_boiler_production_type_steam_or_hw
+            else:
+                steam_or_hw = 'hot_water'
+            chp = CHP(dfm=dfm, run_uuid=run_uuid,
+                      existing_boiler_production_type_steam_or_hw=steam_or_hw,
+                      oa_temp_degF=inputs_dict['Site']['outdoor_air_temp_degF'],
+                      site_elevation_ft=inputs_dict['Site']['elevation_ft'],
+                      time_steps_per_hour=inputs_dict.get('time_steps_per_hour'), **inputs_dict['Site']['CHP'])
+
+            # Update the model inputs if there are calculations with the tech class, CHP.py, etc
+            #tmp = dict()
+            # This is not currently updating anything because tmp is empty (hopefully not doing anything unintended)
+            #ModelManager.updateModel('CHPModel', tmp, run_uuid)
+
+
         util = Util(dfm=dfm,
                     outage_start_hour=inputs_dict['Site']['LoadProfile'].get("outage_start_hour"),
                     outage_end_hour=inputs_dict['Site']['LoadProfile'].get("outage_end_hour"),
                     )
-
+        ##[az] Imported from CHP, not sure if we need?
+        #dfm.add_net_metering(
+        #    net_metering_limit=inputs_dict['Site']['ElectricTariff'].get("net_metering_limit_kw"),
+        #    interconnection_limit=inputs_dict['Site']['ElectricTariff'].get("interconnection_limit_kw")
+        #)
         dfm.finalize()
         dfm_dict = vars(dfm)  # serialize for celery
 
         # delete python objects, which are not serializable
-        for k in ['storage', 'site', 'elec_tariff', 'pvs', 'pvnms', 'load', 'util'] + dfm.available_techs:
+
+        for k in ['storage', 'hot_tes', 'cold_tes', 'site', 'elec_tariff', 'fuel_tariff', 'pvs', 'pvnms'] + dfm.available_techs:
```

Comment:
why were 'util' and 'load' removed from this list?

Reply:
@zolanaj We need to add the two that @NLaws suggested above and 'heating_load' and 'cooling_load'. This is why my local API has not been working! (passing non-serializable objects). Thanks Nick!

Reply:
This was a mistake. In the reformulation, we removed 'load' and 'util' as technologies but upon further review, these are still objects that need to be cleaned up.  This has been fixed in commit 94796777e8dbbf658025b539105506d917b85a49

Reply:
Sorry, to follow up: 'heating_load' and 'cooling_load' have been added to this list in commit 172d85f9903ce14d2f9bf05be43f072c0d31fcc0